### ISSUE TITLE
Force between siblings - Force Directed Tree

### ISSUE THREAD

**Issue by jpnoe** [amcharts/amcharts4#3307]
In force directed tree, is there a way to change the force between siblings rather than just between nodes in general? My goal is to have nodes in the same branch group together while being repelled from the center.

Side note: I have messed around with centerStrength and manyBodyStrength as well as the other various physics settings. manyBodyStrength works as expected, and I was hoping to use centerStrength to repel nodes from the center, but if I have centerStrength at anything other than 0 the nodes begin flying all over the screen.

**Comment by martynasma** [comment]
There's no specific setting that would just affect siblings.

There's a `linkWithStrength` for nodes connected with `linkWith`, but that's probably not what you're looking for.

**Comment by jpnoe** [comment]
Yeah, I didn’t think I was in luck. I did manage to find a remedy by fixing the nodes upon being clicked, since I ran into strange behavior with the nodes jumping out upon toggling them. Not sure if it’s due to my settings or the sheer amount of nodes that are going into the physics calculations (around 100 total on a branch, including all the way down to leaf nodes). 

Anyways, thanks for the quick answer! Y’all are on top of things!
